6-bash-scripting.sh

```diff
@@ -0,0 +1,20 @@
+touch my_file
+chmod +x my_file
+#put these commands inside my_file
+
+#!/bin/bash
+today_date=$(date +%F)
+mkdir "zero_${today_date}_files"
+
+args=${1:-50}
+for i in $(seq $args)
+do
+touch zero_${today_date}_files/FILE_${i}_.dat
+for j in $(seq ${i})
+do
+echo "0" >> zero_${today_date}_files/FILE_${i}_.dat
```

Comment:
Notice that the demand was to save nullbytes (`\x00`) in the file and not ascii char 0 (`\x30`)